pkg/util/k8sutil/self_hosted.go

```diff
@@ -61,11 +62,12 @@ func PodWithAddMemberInitContainer(p *v1.Pod, endpoints []string, name string, p
 	return p
 }
 
-func NewSelfHostedEtcdPod(name string, initialCluster []string, clusterName, state, token string, cs spec.ClusterSpec, owner metatypes.OwnerReference) *v1.Pod {
+func NewSelfHostedEtcdPod(name string, initialCluster []string, clusterName, ns, state, token string, cs spec.ClusterSpec, owner metatypes.OwnerReference) *v1.Pod {
+	selfHostedDataDir := path.Join(etcdVolumeMountDir, ns, clusterName)
```

Comment:
Right. We should probably use member name as guarding.
I prefer `$ns/$cluster/$member` if so.